packages/@react-aria/tag/docs/useTagGroup.mdx

```diff
@@ -0,0 +1,297 @@
+        ))}
+      </div>
+      {description && (
+        <div {...descriptionProps} className="description">
+          {description}
+        </div>
+      )}
+      {errorMessage && validationState === "invalid" && (
+        <div {...errorMessageProps} className="error-message">
+          {errorMessage}
+        </div>
+      )}
+    </div>
+  );
+}
+
+function Tag(props) {
+  let { item, state, allowsRemoving, onRemove } = props;
+  let ref = React.useRef(null);
+  let { isFocused, focusProps } = useFocusRing({
+    within: true
+  });
+
+  let { clearButtonProps, labelProps, gridCellProps, rowProps } = useTag(
+    { ...props, isFocused, item, allowsRemoving, onRemove },
+    state,
+    ref
+  );
+
+  return (
+    <span
+      ref={ref}
+      {...rowProps}
+      {...focusProps}
+    >
+      <span {...gridCellProps} {...labelProps}>{item.rendered}</span>
+      {allowsRemoving && 
+        <Button {...clearButtonProps}>❎</Button>}
+    </span>
+  );
+}
+
+<TagGroup label="Categories">
+  <Item key="news">News</Item>
+  <Item key="travel">Travel</Item>
+  <Item key="gaming">Gaming</Item>
+  <Item key="shopping">Shopping</Item>
+</TagGroup>
+```
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+/* css */
+.tag-group {
+  display: flex;
+  flex-direction: column;
+}
+
+.tag-group div {
+  margin: 5px 0;
+}
+
+.tag-group [role="grid"] {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.tag-group [role="row"] {
+  display: flex;
+  align-items: center;
+  border: 1px solid gray;
+  border-radius: 4px;
+  padding: 2px 5px;
+  margin: 2px;
+}
+
+.tag-group [role="gridcell"] {
+  margin: 0 5px;
+}
+
+.tag-group [role="row"] button {
+  background: none;
+  border: none;
+}
+
+.tag-group .description {
+  font-size: 12px;
+}
+
+.tag-group .error-message {
+  color: red;
+  font-size: 12px;
+}
+```
+
+</details>
+
+### Button
+
+The `Button` component is used in the above example to remove a tag. It is built using the [useButton](useButton.html) hook, and can be shared with many other components.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+import {useButton} from '@react-aria/button';
+
+function Button(props) {
+  let ref = React.useRef(null);
+  let {buttonProps} = useButton(props, ref);
+  return <button {...buttonProps} ref={ref}>{props.children}</button>;
+}
+```
+
+</details>
+
+
+## Styled examples
+
+<ExampleCard
+  url="https://codesandbox.io/s/usetaggroup-with-tailwind-css-zxxrpv"
+  preview={tailwindExample}
+  title="Tailwind CSS"
+  description="A TagGroup built with Tailwind." />
+
+## Usage
+
+### Remove tags
+
+The `onRemove` handler and `allowsRemoving` props can be used to include a clear button which can be used to remove a tag. This allows the user to press the clear button, or press the backspace key while the tag is focused to remove the tag from the group.
```

Comment:
Space key works to remove the tag too, but isn't mentioned here, should it be?

Reply:
This is a good point, although outside of this docs PR.

I could see the argument that we keep it to Delete and Backspace right now, so that space is reserved for selection.

However, I don't know if selection and removal will be supported at the same time.

I can bring this up with the team.